packages/extension-ui/src/Popup/ImportSeed/index.tsx

```diff
@@ -64,7 +63,18 @@ function ImportSeed(): React.ReactElement {
 
   const _onPreviousStep = useCallback(() => setStep((step) => step - 1), []);
 
-  const _onChangeNetwork = useCallback((newGenesisHash: string) => setGenesis(newGenesisHash), []);
+  const _onChangeNetwork = useCallback(
+    (newGenesisHash: string) => {
+      if (!account) {
+        return;
+      }
+
+      setAccount({ ...account, genesis: newGenesisHash });
```

Comment:
Maybe simpler just to
```suggestion
      if (account) {
        setAccount({ ...account, genesis: newGenesisHash });
      }
```

Reply:
I'd leave it as it is, if you don't mind :pray:

Reply:
Sure 👍